src/esp/sim/Simulator.cpp

```diff
@@ -143,28 +145,25 @@ void Simulator::reconfigure(const SimulatorConfiguration& cfg) {
         flags |= gfx::Renderer::Flag::NoTextures;
       renderer_ = gfx::Renderer::create(flags);
     }
-
-    // (re) create scene instance
-    success = createSceneInstance(config_.activeSceneName);
   } else {
-    // (re) create scene instance without renderer
-    success = createSceneInstanceNoRenderer(config_.activeSceneName);
+    CORRADE_ASSERT(!Magnum::GL::Context::hasCurrent(),
+                   "::reconfigure : Unexpected existing context when "
+                   "createRenderer==false", );
   }
 
+  // (re) create scene instance
+  success = createSceneInstance(config_.activeSceneName);
+
   LOG(INFO) << "::reconfigure : createSceneInstance success == "
             << (success ? "true" : "false")
-            << " for active scene name : " << config_.activeSceneName
-            << (config_.createRenderer ? " with" : " without") << " renderer.";
```

Comment:
Why is this info removed from the output?

Reply:
Oops, I think this was an accident from resolving merge conflicts. I'll add it back in.